internal/common/metrics/constants.go

```diff
@@ -114,4 +114,6 @@ const (
 	MemoryUsedHeap  = CadenceMetricsPrefix + "memory-used-heap"
 	MemoryUsedStack = CadenceMetricsPrefix + "memory-used-stack"
 	NumGoRoutines   = CadenceMetricsPrefix + "num-go-routines"
+
+	PollerRemainingBuffer = CadenceMetricsPrefix + "poller-remaining-buffer"
```

Comment:
metrics name is confusing. The terminology we use is "max concurrent task" in worker options. How about having two metrics:
1. concurrent-task-quota
2. concurrent-task-running (or concurrent-task-running)